docs/meetings/client/2022_12_02.md

```diff
@@ -0,0 +1,59 @@
+# Client Meeting Minutes
+
+- **Start Time:** 4:00 PM
+- **End Time:** 4:25 PM
+- **Location:** Zoom
+- **Date:** December 2, 2022
+
+## Attendance
+
+- Client:
+  - Nazim Ragimov (CEO)
+  - Erik (dev)
+- Team A:
+  - Veronica Jack
+  - Edouard Eltherington (laptop died; on mobile, audio only)
+  - Logan Parker
+  - Matthew Kuelker
+
+### Attendance Comments
+
+- Other teams present (unclear if all team members were present)
+  - Team B
+  - Team C
+
+## Team A
+
+- Quick overview of the week:
+  - we have great feedback from our peer testing sessions so we have some stuff to improve
+
+- our goals for next semester will be animating the face and body, along with improved realism
+- our question about making the program a package or something instead of API
+  - Erik: whatever works for us, as long as it can integrate with their system
+  - Logan: sending over HTTP, it might be slower with the audio and text files, so to avoid additional time sending...
+  - will stick with original plan of API
+- determine when we'll message them in January to determine the new meeting schedule
+
+## Team B
+
+- this week, they were working on their animator request flushed out
+- will send the word document outlining what frames are needed (mouths and poses), animator notes today
+- 22 different mouths, many may overlap with other teams
+- same characters, or easy colour changes
+- API is almost done, needs to be uploaded
+
+## Team C
+
+- does Kukarella have a virtual private server to test on?
+  - Erik: maybe? ask Group B what they did or try a cloud service for free trial, or just have local API working. It

```

Comment:
sounded like basically a no, I'm under the impression they're interested in keeping their systems separate from ours until the end, and handling implementation themselves

Reply:
Yeah, I think it was more a no as well. I'll adjust